pkg/operation/seed/scheduler/gardener_kube_scheduler.go

```diff
@@ -0,0 +1,367 @@
+									Requests: corev1.ResourceList{
+										corev1.ResourceCPU:    resource.MustParse("23m"),
+										corev1.ResourceMemory: resource.MustParse("64Mi"),
+									},
+									Limits: corev1.ResourceList{
+										corev1.ResourceCPU:    resource.MustParse("400m"),
+										corev1.ResourceMemory: resource.MustParse("512Mi"),
+									},
+								},
+								VolumeMounts: []corev1.VolumeMount{
+									{
+										Name:      configVolumeName,
+										MountPath: volumeMountPathConfig,
+									},
+								},
+							},
+						},
+						Volumes: []corev1.Volume{
+							{
+								Name: configVolumeName,
+								VolumeSource: corev1.VolumeSource{
+									ConfigMap: &corev1.ConfigMapVolumeSource{
+										LocalObjectReference: corev1.LocalObjectReference{
+											Name: configMap.Name,
+										},
+									},
+								},
+							},
+						},
+					},
+				},
+			},
+		}
+		serviceAccount = &corev1.ServiceAccount{ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: k.namespace,
+			Labels:    getLabels(),
+		}}
+		roleBinding = &rbacv1.RoleBinding{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      roleBindingName,
+				Namespace: metav1.NamespaceSystem,
+				Labels:    getLabels(),
+			},
+			RoleRef: rbacv1.RoleRef{
+				APIGroup: rbacv1.GroupName,
+				Kind:     "Role",
+				Name:     roleName,
+			},
+			Subjects: []rbacv1.Subject{{
+				Kind:      rbacv1.ServiceAccountKind,
+				Name:      name,
+				Namespace: k.namespace,
+			}},
+		}
+		webhook = &admissionregistrationv1beta1.MutatingWebhookConfiguration{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:   webhookName,
+				Labels: getLabels(),
+			},
+			Webhooks: []admissionregistrationv1beta1.MutatingWebhook{{
+				Name:         webhookName,
+				ClientConfig: *k.webhookClientConfig,
+				Rules: []admissionregistrationv1beta1.RuleWithOperations{{
+					Operations: []admissionregistrationv1beta1.OperationType{admissionregistrationv1beta1.Create},
+					Rule: admissionregistrationv1beta1.Rule{
+						APIGroups:   []string{corev1.GroupName},
+						APIVersions: []string{corev1.SchemeGroupVersion.Version},
+						Scope:       &scope,
+						Resources:   []string{"pods"},
+					},
+				}},
+				FailurePolicy: &failPolicy,
+				MatchPolicy:   &matchPolicy,
+				NamespaceSelector: &metav1.LabelSelector{
+					MatchLabels: map[string]string{
+						v1beta1constants.GardenRole: v1beta1constants.GardenRoleShoot,
+					},
+				},
+				ObjectSelector:          &metav1.LabelSelector{},
+				SideEffects:             &sideEffects,
+				TimeoutSeconds:          &timeout,
+				AdmissionReviewVersions: []string{admissionregistrationv1beta1.SchemeGroupVersion.Version},
+				ReinvocationPolicy:      &revocationPolicy,
+			}},
+		}
+		registry = managedresources.NewRegistry(kubernetes.SeedScheme, kubernetes.SeedCodec, kubernetes.SeedSerializer)
+	)
+
+	if _, err := controllerutil.CreateOrUpdate(ctx, k.client, namespace, func() error {
+		namespace.Labels = getLabels()
```

Comment:
Hm, if this is an existing namespace then you'd overwrite all labels here - probably it's better to merge them instead?